calendar-app/Screens/ProfileScreen.js

```diff
@@ -0,0 +1,101 @@
+import { GlobalColor, GlobalTextColor } from '../Styles';
+import { storeUserInfo, getUserId, getGroupId } from '../Components/Storage/userDataStorage';
+import ButtonComp from '../Components/ButtonComp';
+import Header from '../Components/Header';
+import { useEffect, useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native'; 
+import { getGroupName } from '../Components/API/Groups/GroupName';
+import { getInGroupStatus } from '../Components/Storage/userDataStorage';
+import { getUserById } from '../Components/API/Users/UserGetById';
+
+const ProfileScreen = () => {
+    const [groupNameData, setGroupName] = useState('');
+    const [inGroup, setInGroup] = useState(null);
+    const [username, setUsername] = useState('');
+
+    async function getGroupStatus() {
+        const inGroup = await getInGroupStatus();
+        return inGroup;
+    };
+    const fetchGroupName = async () => {
+        const groupId = await getGroupId();
+        const groupNameData = await getGroupName({ groupId });
+        setGroupName(groupNameData);
+    };
+
+    async function fetchUsername() {
+        const userId = await getUserId();
+        const user = await getUserById({ userId });
+        if (user) {
+            setUsername(user.username);
+        }
+    }
+
+
+    useEffect(() => {
+        fetchUsername();
+        fetchGroupName();
+        getGroupStatus().then(setInGroup);
+    }, []);
+
+    return (
+        <View style={styles.wrapper}>
+            <Header title={"Profile"} />
+            <View style={styles.container}>
+                <Text style={styles.topLine}>About You</Text>
+                <View style={styles.segment}>
+                    <Text style={styles.value}>Username:</Text>
+                    <Text style={styles.value}>{username}</Text>
+                </View>
+                <View style={styles.segment}>
+                    <Text style={styles.value}>Group:</Text>   
+                    {
+                        inGroup ? 
+                        <Text style={styles.value}>{groupNameData.name}</Text> 
+                        : <Text style={styles.value}>Not in a group</Text>
+                    }    
+                </View>
+                <View style={styles.segment}>
+                    <Text style={styles.value}>Email:</Text>
+                    <Text style={styles.value}>----</Text>
```

Comment:
email is hardcoded, the best choice is to make it a placeholder.

Reply:
It matches Jira requirements